### ISSUE TITLE
Ports.h

### ISSUE THREAD

**Issue by TylerSeiford** [FRCTeam2501/2020-Robot#12]
1. JOYSTICK should only have driver and control sticks.
2. PWMs should all be empty.
3. CAN should have INTAKE_LEFT and INTAKE_RIGHT, and then EMPTY_8 to EMPTY_11
4. CAN should also have a CAN_PDP of 0
5. DIOs should all be empty.
6. PCM should have a solenoid for the hopper pins

**Comment by TylerSeiford** [comment]
Nice work Brian, https://github.com/FRCTeam2501/2020-Robot/commit/622488c5a9201e019ee18efc1599e85eb470f250 fixes most of these.

Issue 3 still needs to be addressed (currently only one `INTAKE` is in the ports.h`.

The DIO_PORTs should have `EMPTY_X` for X from 0 to 9 instead of a single empty.

The PCM still needs ports for the hopper pins.




In addition, we learned last night that we can not flash a SparkMAX as CAN ID of 0, so we needed to redo `LEFT_FRONT` as ID 16, and then we need to replace the current `LEFT_FRONT` with an `EMPTY_0`.  We also will need a new namespace for PDP ports, something of this form:
```cpp

namespace PDP {
	enum PDP_PORTS {
		LEFT_FRONT = 0,
		LEFT_REAR,
		RIGHT_FRONT,
		RIGHT_REAR,
		INTAKE_LEFT,
		INTAKE_RIGHT,
		HOPPER_LEFT,
		HOPPER_RIGHT,
		PCM,
		EMPTY_9,
		EMPTY_10,
		EMPTY_11,
		CLIMBER_LEFT,
		CLIMBER_RIGHT,
		SHOOTER_LEFT,
		SHOOTER_RIGHT
	};
}

```

That way we can map the PDP ports to motor controllers easier (for later features).
